srcpkgs/onionshare-desktop/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'onionshare-desktop'
+pkgname=onionshare-desktop
+version=2.4
+revision=1
+wrksrc="onionshare-${version}"
+build_style=python3-module
+pycompile_dirs="/usr/share/nautilus-python/extensions"
+hostmakedepends="python3-setuptools"
+depends="python3-Flask-HTTPAuth python3-Flask-SocketIO python3-pycryptodome python3-PyQt5
+ python3-pysocks python3-requests onionshare-cli python3-pyside2 python3-stem tor"
+short_desc="Share files anonymously and securely"
+maintainer="DragonGhost7 <darkiridiumghost@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://onionshare.org/"
+changelog="https://raw.githubusercontent.com/micahflee/onionshare/develop/CHANGELOG.md"
+distfiles="https://github.com/micahflee/onionshare/archive/v${version}.tar.gz"
```

Comment:
onionshare-cli and onionshare-desktop have the same distfiles, so they should either be built in the same template or have a comment on top to remind people updating this package to keep the versions synchronized between both templates.

Reply:
yeah i think best would be to make onionshare-desktop a subpackage of onionshare, managinf the 2 will be much easier

Reply:
but they have separate builds, and installs, so might be complicated